firestore-bigquery-export/firestore-bigquery-change-tracker/package.json

```diff
@@ -10,7 +10,7 @@
   "main": "./lib/index.js",
   "scripts": {
     "build": "npm run clean && npm run compile",
-    "clean": "rimraf lib",
+    "clean": "rimraf lib/",
```

Comment:
Was this change necessary, should be working without this?

Reply:
Trying to make it more explicit it's a directory, but I'll change it back.